### DISCUSSION TITLE
TCP traffic split visualization on Kiali graph in percentage

### DISCUSSION THREAD

**Discussion by ravikumarpandit92** [kiali/kiali#5795] [Community meetings]
Hello experts,
**Issue :**  I am trying to visualize the traffic split percentage on Kiali graph. I am able to see only http type call in percentage but not for TCP.
**Que :**   Is there any additional configuration has to be done for visualizing the percentage wise traffic split for TCP.

**Comment by jmazzitelli** [comment]
_converted to a Q&A discussion_

**Comment by jshaughn** [comment]
The problem is that TCP traffic is measure in bytes, not requests.  This is why when you turn on rate edge labels you see "rps/RequestsPerSecond" for HTTP edges and "bps/BytesPerSecond" for TCP edges.   Kiali generates the request percentages because each request basically has equal weight (i.e. 1),  But for TCP the weights, being payload sizes, can vary widely, so the thinking was that it wasn't going to be very useful.
